package.json

```diff
@@ -24,14 +24,14 @@
     "babel-plugin-transform-class-properties": "^6.24.1",
     "babel-plugin-transform-decorators-legacy": "^1.3.4",
     "ember-cli-babel": "^6.0.0",
+    "ember-cli-dependency-checker": "^2.0.0",
```

Comment:
Ah, totally confused those two. That explains why it was the wrong version, some other addon I'm using must have included a previous one (I was wondering how that would have happened)